watchMeTests/forwardingRewindingVideo.test.js

```diff
@@ -0,0 +1,101 @@
+const {
+	snippetHomepageOpened,
+	snippetOpenVideo} = require('./common');
+const {forwardBtnFocused, pausedBtnFocused, rewindBtnFocused} = require('./elements');
+const {snapshotElement} = require('./utils');
+
+const suitest = require('suitest-js-api');
+const {assert, VRC, PROP, COMP, VALUE, VIDEO_STATE} = suitest;
+
+describe('Forwarding/Rewinding video', async() => {
+	before(async() => {
+		await assert.openApp();
+		await snippetHomepageOpened();
+		/**
+		 * Navigate to the video and open it using the snippet test.
+		 */
+		await snippetOpenVideo();
+	});
+
+	it('forwarding rewinding video', async() => {
+		await assert.video().matches([
+			{
+				name: PROP.VIDEO_LENGTH,
+				val: 1810,
+				type: COMP.GREATER,
+			},
+			{
+				name: PROP.VIDEO_POSITION,
+				val: 1000,
+				type: COMP.GREATER,
+			},
+			{
+				name: PROP.VIDEO_STATE,
+				val: VIDEO_STATE.PLAYING,
+			},
+			{
+				name: PROP.VIDEO_URL,
+				val: 'http://file.suite.st/sampleapp/api/video/a84d67e5-c6d4-41c0-9f70-cd8c7a5f6f4f.mp4',
+			},
+		]).timeout(10000);
+		/**
+		 * Focus the "Forward" button on the control panel.
+		 */
+		await assert.press(VRC.DOWN);
+		await snapshotElement(pausedBtnFocused).timeout(2000);
```

Comment:
Timeout is 2s by default in JS API, so it's not really needed to specify it explicitly.

Reply:
> Timeout is 2s by default in JS API, so it's not really needed to specify it explicitly.

@andriichumak it actually was changed to 1ms `const DEFAULT_DELAY = 1;`. I'm not exactly sure why thought.

Reply:
With browser it works

Reply:
@givehug , delay for the button presses was changed, but not a timeout for assertions

Reply:
thats right